pytorch_lightning/utilities/types.py

```diff
@@ -0,0 +1,8 @@
+from typing import Dict, Iterator, Optional, Union
+
+import torch
+from torchmetrics import Metric
+
+_METRIC = Union[Metric, torch.Tensor, int, float]
+_STEP_OUTPUT = Optional[Union[torch.Tensor, Dict[str, torch.Tensor]]]
```

Comment:
step outputs can be anything inside the dict, it's not restricted to tensor, right?